tendrl/monitoring_integration/sync/gluster_cluster_details.py

```diff
@@ -14,31 +14,24 @@
 }
 
 
-def get_cluster_details(integration_id):
-    cluster_details = {}
+def get_cluster_details(integration_id, cluster_details):
```

Comment:
Don't pass `cluster_details` as parameter. Rather create a dict inside function and return the same.